app/views/posts/show.html.erb

```diff
@@ -9,5 +9,18 @@
         <%= @post.text %>
     </div>
 
+    <div class="comment-container">
+        <%= form_with model: @comment, url: create_new_comment_path(user_id: @user.id, id: @post.id) ,class: "flex items-start gap-1" do |form| %>
+            <%= form.text_area :text, size: "70x5" %>
+            <%= form.submit "Add new comment" %>
+        <% end %>
+    </div>
+
+   <div class="comment-container">
+  <%= form_with model: @like, url: user_post_likes_path(@user, @post), method: :post, class: "flex items-start gap-1" do |form| %>
+    <%= form.submit "Like", class: "w-full" %>
+  <% end %>
+</div>
```

Comment:
- [ ] There's also a bug when trying to create a comment for a post (I've manually created a post using the CLI in this case) once again the bug shows an undefined route "create_new_comment_path" exists from the above highlighted code so please make sure to work on this.
  ![image](https://github.com/abdozayan12/DataApp/assets/104968236/611b99c3-7b37-41f5-b8a2-0b62895a3a32)